src/Microsoft.AspNetCore.OData/Microsoft.AspNetCore.OData.csproj

```diff
@@ -44,10 +44,13 @@
     <PackageReference Include="Microsoft.OData.Core" Version="7.12.1" />
     <PackageReference Include="Microsoft.OData.Edm" Version="7.12.1" />
     <PackageReference Include="Microsoft.Spatial" Version="7.12.1" />
+    <PackageReference Include="Microsoft.OData.Core" Version="7.11.1" />
```

Comment:
Why are you referencing both 7.11.1 and 7.12.1 versions of Microsoft.OData.Core?

Reply:
I have seen this duplication whenever I rebase the feature branch. Needs manual cleanup